sdks/python/apache_beam/dataframe/frames_test.py

```diff
@@ -204,22 +239,6 @@ def new_column(df):
     })
     self._run_inplace_test(new_column, df)
 
-  def test_str_split(self):
-    s = pd.Series([
-        "this is a regular sentence",
-        "https://docs.python.org/3/tutorial/index.html",
-        np.nan
-    ])
-
-    # TODO(BEAM-11931): pandas produces None for empty values with expand=True,
```

Comment:
Adding a note to myself that this jira should be closed as obsolete when this PR is merged